lab_3_ann_retriever/main.py

```diff
@@ -91,6 +104,9 @@ def __init__(self, stop_words: list[str]) -> None:
         Args:
             stop_words (list[str]): List with stop words
         """
+        self.documents = []
```

Comment:
there is not `self` inside this function. It is not a method.

Reply:
changed